### PR TITLE
Only install strict dependencies

### PR DESCRIPTION
**Pull request by mgax**
This patch skips any suggested or recommended packages to keep our image as small as possible.

### CODE REVIEW

**Comment by mgax** [comment]
I started a [prerequisites build](https://jenkins.liquiddemo.org/job/setup-prerequisites/job/apt-strict-deps/2/), let's see how it does before merging.

**Comment by mgax** [comment]
[Build #3](https://jenkins.liquiddemo.org/job/setup-prerequisites/job/apt-strict-deps/3/) passed, and the resulting image is a bit smaller (1.15GB instead of 1.22GB) and builds a bit quicker (19 min instead of 23 min). Ready to merge.

File: ansible/roles/common/tasks/main.yml [hunk 1]
```diff
@@ -45,7 +45,7 @@
     - supervisor
     - nginx
     - python3
-    - python-virtualenv
+    - virtualenv
```
Comment (gabriel-v) [inline ansible/roles/common/tasks/main.yml]: `$ apt-cache show virtualenv` says:
```
Replaces: python-virtualenv (<< 1.11.6)
Depends: python3, python3-virtualenv
```

So yeah, good idea

File: py/images/builders/base.py [hunk 2]
```diff
@@ -77,6 +77,11 @@ def open_target(self, image):
                     yield target
 
     def prepare_chroot(self, target):
+        apt_conf_liquid = target.mount_point / 'etc/apt/apt.conf.d/liquid'
```
Comment (gabriel-v) [inline py/images/builders/base.py]: `liquid@liquid:~$ ls /etc/apt/apt.conf.d/` says: 
```01-vendor-ubuntu  01autoremove  01autoremove-kernels  01autoremove-postgresql  10periodic  15update-stamp  20archive  70debconf  99update-notifier```

So I'd go with naming our file `999-liquid-investigations`, even if `'l' > '9'` in ASCII.

Comment (mgax) [reply py/images/builders/base.py]: I don't think we need to worry about the order, we're the only ones to set those APT options.

**Pull request merged**